src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1030,6 +1030,8 @@ class ApplicationManagerImpl
   bool IsSOStructValid(const hmi_apis::StructIdentifiers::eType struct_id,
                        const smart_objects::SmartObject& display_capabilities);
 
+  virtual void UnsubscribeAppFromSoftButtons(
+      const commands::MessageSharedPtr response) OVERRIDE;
```

Comment:
Please, add blank line between methods

Reply:
@APipko Fixed in 9062a2e6bb5787c23a1e305bcbe7176ae28bfc9e.